test/actor_test.py

```diff
@@ -331,7 +331,7 @@ def __init__(self):
     random.seed(1234)
     f2 = Foo.remote()
 
-    assert f1._ray_actor_id.id() != f2._ray_actor_id.id()
+    assert f1._ray_actor_id.binary() != f2._ray_actor_id.binary()
```

Comment:
We can just compare the IDs directly, right?

Reply:
yeah, changed it